packages/shared/lib/core/wallet/interfaces/send-confirmation-parameters.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { IAsset, Subject } from '@core/wallet'
+
+export interface ISendConfirmationParameters {
+    asset?: IAsset
+    amount?: string | undefined
+    unit?: string
+    recipient?: Subject
+    metadata?: string
+    tag?: string
+    giftStorageDeposit?: boolean
```

Comment:
Does this not need the two other send parameters for disabling the gifting of the deposit, etc.? Might be related to the above comment about the different between "send operation parameters" and "send confirmation operation parameters"?

Reply:
Yeah added them